at_secondary/at_secondary_server/lib/src/server/at_secondary_impl.dart

```diff
@@ -373,6 +373,7 @@ class AtSecondaryServerImpl implements AtSecondaryServer {
     await notificationInstance.init(notificationStoragePath!);
     // Loads the notifications into Map.
     await NotificationUtil.loadNotificationMap();
+    NotificationUtil.scheduleKeyExpireTask(expiringRunFreqMins!, currentAtSign);
```

Comment:
Why a Util? Do the ***Job classed I believe.

Reply:
done